crates/hyperswitch_connectors/src/connectors/fiuu.rs

```diff
@@ -33,20 +35,41 @@ use hyperswitch_interfaces::{
     types::{self, Response},
     webhooks,
 };
-use masking::{ExposeInterface, Mask};
+use reqwest::multipart::Form;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
 use transformers as fiuu;
 
 use crate::{constants::headers, types::ResponseRouterData, utils};
 
+fn parse_response<T>(data: &[u8]) -> Result<T, errors::ConnectorError>
+where
+    T: for<'de> Deserialize<'de>,
+{
+    let response_str = String::from_utf8_lossy(data);
+    let mut map = HashMap::new();
+    for line in response_str.lines() {
+        if let Some((key, value)) = line.split_once('=') {
+            map.insert(key.to_string(), value.to_string());
+        }
+    }
+    let json = serde_json::to_string(&map)
+        .map_err(|_| errors::ConnectorError::ResponseDeserializationFailed)?;
+
+    let response: T = serde_json::from_str(&json)
+        .map_err(|_| errors::ConnectorError::ResponseDeserializationFailed)?;
```

Comment:
1. We may be losing some information because we're calling `from_utf8_lossy()`. 
2. Can we avoid the split by `=` and find a better way to deserialize the response if possible?
3. The 3 step process where we collect key-values to a map, serialize it as JSON, then deserialize it into `T` must preferably be avoided, if we can find a better way to deserialize the response.
4. If we end sticking with this, then at least log the errors, do not ignore them.